README.md

```diff
@@ -1,15 +1,53 @@
 # ForemanPuppet
 
-Adds Puppet External node classification funcionality for [Foreman](theforeman.org).
+[![Gem Version](https://img.shields.io/gem/v/foreman_puppet.svg)](https://rubygems.org/gems/foreman_puppet)
+[![Code Climate](https://codeclimate.com/github/theforeman/foreman_puppet/badges/gpa.svg)](https://codeclimate.com/github/theforeman/foreman_puppet)
+[![GPL License](https://img.shields.io/github/license/theforeman/foreman_ansible.svg)](https://github.com/theforeman/foreman_ansible/blob/master/LICENSE)
 
+This plugin adds Puppet External node classification functionality to [Foreman](theforeman.org).
+
+* Website: [theforeman.org](http://theforeman.org)
+* Support: [Foreman support](http://theforeman.org/support.html)
+
+## Features
+
+* Puppet dashboards
+* Puppet
+    * Environments
+    * Classes
+    * Config groups
+    * Smart Class parameters
+* Host / Hostgroup
+    * Classes assignment
+    * Environment assignment
+    * Puppet proxy assignment
+* Puppet ENC (external node classifier)
+* Smartproxy Puppet status/tab
+* Template hostgroup / environment
+
+## Compatibility
+
+The functionality will be extracted from Foreman 3.0. Therefore, this plugin is only required for 3.0 and onwards.  
```

Comment:
```suggestion
Foreman 3.0 will be the first release where the Puppet functionality is fully extracted. Therefore, this plugin is only required for 3.0 and higher.  
```

I think the *from* makes it sound like it is happening in 3.0 rather than

Reply:
Thanks. I've changed the wording to your version.